templates/conf/offload.env.template.common

```diff
@@ -121,13 +121,6 @@ export OFFLOAD_CONFDIR=${OFFLOAD_HOME}/conf
 # Override log path, defaults to OFFLOAD_HOME/log
 #export OFFLOAD_LOGDIR=
 
-# Default number of offload buckets (subpartitions) for parallel data retrieval
-# An explicit integral value for DEFAULT_BUCKETS disables any automated decision making
-# If you aim to run queries with parallel DOP X then set the DEFAULT_BUCKETS[_MAX] to X
-# this way each Oracle PX slave can start its own connector for fetching a subset of data
-# DEFAULT_BUCKETS=AUTO tunes to a value between 1 and DEFAULT_BUCKETS_MAX
-export DEFAULT_BUCKETS=AUTO
-# Upper limit of DEFAULT_BUCKETS
 export DEFAULT_BUCKETS_MAX=16
```

Comment:
Does this export need to be removed? Or its comment reinstated?

Reply:
Good spot, the export needed to be removed.